Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -65,20 +76,37 @@ public class RoutingTable
         /// <summary>
         /// The number of peers in the table.
         /// </summary>
-        public int Count => _buckets.Sum(bucket => bucket.Count);
+        public int Count => _buckets.Sum(bucket => bucket.Count) + _staticPeerStates.Length;
+
+        /// <summary>
+        /// An <see cref="IEnumerable{T}"/> of static peers in the table.
+        /// </summary>
+        public IEnumerable<BoundPeer> StaticPeers => _staticPeerStates
```

Comment:
How about typing this as `IReadOnlyList<BoundPeer>` instead of `IEnumerable<BoundPeer>` to get rid of [possible multiple enumeration warning][1]?

```suggestion
        public IReadOnlyList<BoundPeer> StaticPeers => _staticPeerStates
```

[1]: https://www.jetbrains.com/help/resharper/PossibleMultipleEnumeration.html